evm_arithmetization/src/generation/jumpdest.rs

```diff
@@ -0,0 +1,191 @@
+use std::cmp::max;
```

Comment:
This is really missing some high-level documentation - what is a jumpdest, why do I care about it, how does it fit into the wider picture?

Reply:
Strong point.  I added some module rustdoc.

Let me know how  you like it.

Reply:
The new doc is failing rustdoc job FYI